nikola/plugins/compile/rest/__init__.py

```diff
@@ -233,7 +241,7 @@ def rst2html(source, source_path=None, source_class=docutils.io.StringInput,
              parser=None, parser_name='restructuredtext', writer=None,
              writer_name='html', settings=None, settings_spec=None,
              settings_overrides=None, config_section=None,
-             enable_exit_status=None, logger=None, l_add_ln=0):
+             enable_exit_status=None, logger=None, l_add_ln=0, site=None):
```

Comment:
It would be better to just pass a `transforms=None` argument.  Also, `getattr` is unnecessary.